src/log.rs

```diff
@@ -81,6 +84,376 @@ type CompatFilterStream<T, R> = Compat01As03<FilterStream<T, R>>;
 /// that can be queried in order to stream logs.
 type CompatCreateFilter<T, R> = Compat01As03<CreateFilter<T, R>>;
 
+/// The default poll interval to use for polling logs from the block chain.
+#[cfg(not(test))]
+pub const DEFAULT_POLL_INTERVAL: Duration = Duration::from_secs(5);
+#[cfg(test)]
+pub const DEFAULT_POLL_INTERVAL: Duration = Duration::from_secs(0);
```

Comment:
This seems kinda weird... Shouldn't test related things be declared within the tests?

Reply:
This is just saying that the poll interval is 0 when running in test mode, so that futures resolve immediately for unit tests.